### PR TITLE
Better HTTP_UNAUTHORIZED message.

### PR DESCRIPTION
**Pull request by benletchford**
Spent about 2 hours pulling my hair out as I was unauthorized. Turns out my account was under au.bigml.io. The error message wasn't helpful. Hopefully this will help someone in the future.

### CODE REVIEW

File: bigml/bigmlconnection.py [hunk 1]
```diff
@@ -783,7 +783,7 @@ def error_message(self, resource, resource_type='resource', method=None):
                         resource_type, resource_type))
                 return error
             if code == HTTP_UNAUTHORIZED:
-                error += u'\nDouble-check your credentials, please.'
+                error += u'\nDouble-check your credentials and the general domain your account is registered with, please.'
```
Comment (mmerce) [inline bigml/bigmlconnection.py]: Thanks for the feedback! We try to comply with the 79 column limit to ensure code readability. Could you please adapt your change to this requirement? You could also add the information about the domain it is currently pointing to. Maybe something like:

```python
                error += u'\nDouble-check your credentials and the general'
                         u' domain your account is registered with, please'
                         u' (currently using %s).' % self.general_domain
```

Comment (benletchford) [inline bigml/bigmlconnection.py]: Thanks for the quick comment. Done!

**Comment by mmerce** [comment]
@benletchford: your PR has been merged, thanks for your contribution!

**Comment by benletchford** [comment]
Thanks!

**Pull request merged**